apps/api/tests/test_cart.py

```
@@ -0,0 +1,365 @@
+
+        assert response.status_code in [401, 404]
+
+    def test_get_cart_with_total_calculation(self, client, user_headers):
+        """Deve calcular total do carrinho"""
+        response = client.get('/api/cart', headers=user_headers)
+
+        if response.status_code == 200:
+            data = response.get_json()
+            if isinstance(data, dict):
+                # Pode ter campos como 'total', 'subtotal', 'tax'
+                has_total = any(key in data for key in ['total', 'subtotal', 'total_amount'])
```

Ruff (0.14.5)

Local variable `has_total` is assigned to but never used

Remove assignment to unused variable `has_total`

(F841)